frappe/public/js/frappe/ui/page.js

```diff
@@ -381,9 +381,15 @@ frappe.ui.Page = Class.extend({
 			}
 
 		} else {
-			return $('<button class="btn btn-default btn-xs" style="margin-left: 10px;">'+__(label)+'</btn>')
-				.on("click", _action)
-				.appendTo(this.inner_toolbar.removeClass("hide"));
+			var button = this.inner_toolbar.find('button')
```

Comment:
I don't think it is a good idea to make `add_inner_button ` take so many decisions. It should be fixed at it's usage.